src/main/jbake/content/jsonb.adoc

```diff
@@ -10,18 +10,18 @@ prev=jsonp008.html
 21 JSON Binding
 ---------------
 
-This chapter describes the Java API for JSON Binding (JSON-B). JSON is a data exchange format widely used
+This chapter describes the Jakarta JSON Binding. JSON is a data exchange format widely used
 in web services and other connected applications. For a brief overview of JSON, see
 link:jsonp001.html#introduction-to-json[Introduction to JSON].
 
-The link:https://jcp.org/en/jsr/detail?id=367[JSR 367] specification provides a standard
-binding layer (metadata and runtime) between Java classes and JSON documents. The JSON-B reference
+The link:https://jakarta.ee/specifications/json-binding/[Jakarta JSON Binding] specification provides a standard
+binding layer (metadata and runtime) between Java classes and JSON documents. One Jakarta JSON Binding reference
 implementation is Yasson, which is developed through Eclipse.org and is included as part of GlassFish Server.
-You can learn more about Yasson at https://projects.eclipse.org/projects/rt.yasson.
+You can learn more about Yasson at https://projects.eclipse.org/projects/ee4j.yasson.
 
 The following topics are addressed here:
 
 * link:jsonb001.html#json-binding-in-the-jakarta-ee-platform[JSON Binding in the Jakarta EE Platform]
 * link:jsonb002.html#overview-of-the-json-binding-api[Overview of the JSON Binding API]
 * link:jsonb003.html#running-the-jsonbbasics-example-application[Running the jsonbbasics Example Application]
-* link:jsonb004.html#further-information-about-the-java-api-for-json-binding[Further Information about the Java API for JSON Binding]
+* link:jsonb004.html#further-information-about-the-java-api-for-json-binding[Further Information about the Jakarta JSON Binding]
```

Comment:
Don't need "the": "the Jakarta" -> "Jakarta"

Update the tag name as well:
further-information-about-jakarta-json-binding

Reply:
Done with the "the". Regarding the tag name, I prefer to do it later if you don't mind, as explained at https://github.com/eclipse-ee4j/jakartaee-tutorial/pull/159#discussion_r324287923